scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+
+#### gVCF annotations
+
+Input gVCF files for the GVS workflow must include the annotations described in the table below:
+
+| Annotation | Description | Notes |
+| --- | --- | --- |
+| Ref | Reference allele | --- |
+| Alt | Alternate allele | --- |
+| AS_RAW_MQ, RAW_MQandDP, or RAW_MQ | RMS mapping quality (‘AS’: allele-specific) | Required for VQSR Data |
+| AS_RAW_MQRankSum or Map_QUAL_RANK_SUM_KEY | Z-score from Wilcoxon rank sum test of alternate versus reference read mapping qualities | Required for VQSR Data |
+| QUALapprox | Sum of PL[0] values; used to approximate the QUAL score | Required for VQSR Data |
+| AS_QUALapprox | Allele-specific sum of PL[0] values; used to approximate the QUAL score | Required for VQSR Data |
+| AS_SB_TABLE or STRAND_BIAS_BY_SAMPLE | Allele-specific forward/reverse read counts for strand bias tests | Required for VQSR Data |
+| AS_VarDP, VarDP, or DP | Depth over variant genotypes, or read depth  (‘AS’: allele-specific) | Required for VQSR Data |
+| call_GT | Genotype | --- |
+| call_GQ | Genotype quality | --- |
+
+### Inputs
+
+The GVS workflow inputs are specified in the Terra GVS beta workspace [workflow configuration](LINK).
+
+#### Sample data input
+
+The GVS workflow takes in reblocked single sample gVCF files and their corresponding index files as `input_vcfs` and `input_vcf_indexes`, respectively. While the GVS workflow takes in up to 100,000 single sample gVCF files as input, only datasets of up to 10,000 files are being used for beta testing.
+
+The [Terra GVS beta workspace](LINK) has been configured with example reblocked gVCF files that you can use to test the workflow. To test the workflow on your own sample data, you’ll need to create an additional data table in your workspace that specifies the locations of the files in the cloud. For step-by-step instructions on creating data tables in Terra, see [How to make a data table from scratch or a template](https://support.terra.bio/hc/en-us/articles/6197368140955). If your data is stored locally, you can upload it to your workspace bucket using the Data Uploader in Terra by following the instructions in the article, [How to use the Data Uploader](https://support.terra.bio/hc/en-us/articles/4419428208411). You’ll also need to ensure that you have a BigQuery dataset and Google project with the correct permissions and that your gVCF files are reblocked, as described in the [Workflow requirements](#workflow-requirements) section above.
+
+#### Input descriptions
+
+The table below describes the GVS workflow input variables:
+
+| Input variable name | Description | Type |
+| --- | --- | --- |
+| dataset_name | String describing the name of the BigQuery dataset. | String |
```

Comment:
Will the data files specified in the input be added to this dataset? I'm new to use BigQuery as part of a workflow, so I'm just curious for more details on how this BigQuery dataset is used relative to the sample inputs.

Reply:
a BQ dataset is a collection of tables. The data from the gvcf files will be added into specific (vet and ref_ranges) tables in the dataset.